### ISSUE TITLE
Uncaught Error: Hostname/IP doesn't match certificate's altnames: "Host: github.com. is not in the cert's altnames: IP Address:172.31.1.251"

### ISSUE THREAD

**Issue by ishaand** [atom/atom#5034]
[Enter steps to reproduce below:]

1. atom --safe
2. ...

**Atom Version**: 0.170.0
**System**: linux 3.16.2-031602-generic
**Thrown From**: Atom Core

### Stack Trace

Uncaught Error: Hostname/IP doesn't match certificate's altnames: "Host: github.com. is not in the cert's altnames: IP Address:172.31.1.251"

```
At events.js:85

Error: Hostname/IP doesn't match certificate's altnames: "Host: github.com. is not in the cert's altnames: IP Address:172.31.1.251"
  at Object.checkServerIdentity (tls.js:209:15)
  at TLSSocket.<anonymous> (_tls_wrap.js:919:31)
  at TLSSocket.emit (events.js:104:17)
  at TLSSocket._finishInit (_tls_wrap.js:457:8)

```

### Commands

```
```

### Config

```json
{
  "core": {
    "themes": [
      "atom-dark-ui",
      "atom-dark-syntax"
    ]
  },
  "editor": {
    "invisibles": {}
  }
}
```

### Installed Packages

```coffee
# User
No installed packages

# Dev
No dev packages
```

**Comment by ishaand** [comment]
See https://github.com/atom/atom/issues/4974

**Comment by kevinsawicki** [comment]
Was the settings view open when this happened?

**Comment by kevinsawicki** [comment]
I believe this was fixed by https://github.com/atom/settings-view/commit/3fe0db5dc1c7db9f936b506a126d689057a24e0d which was included in the latest release, 0.172
